.circleci/config.yml

```
@@ -0,0 +1,87 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+# Settings common to each job
+var_job_defaults: &job_defaults
+  working_directory: ~/verdaccio-google-cloud-storage
+  docker:
+    - image: cimg/node:lts-browsers
+
+orbs:
+  node: circleci/node@5.1.0
+  build-tools: circleci/build-tools@3.0.0
+
+commands:
+  # Command for checking out the source code from GitHub. This also ensures that the source code
+  # can be merged to the master branch without conflicts.
+  checkout_and_rebase:
```

Let's use kebab-case for consistency:

```suggestion
  checkout-and-rebase:
```